llvm/include/llvm/Analysis/ValueTracking.h

```diff
@@ -330,6 +330,13 @@ bool isKnownNeverNaN(const Value *V, unsigned Depth, const SimplifyQuery &SQ);
 std::optional<bool> computeKnownFPSignBit(const Value *V, unsigned Depth,
                                           const SimplifyQuery &SQ);
 
+/// Return true if the sign bit of result can be ignored when the result is
+/// zero.
```

Comment:
I think from a ValueTracking perspective, these APIs should check whether a given `Use &` can ignore the sign bit. The calling code should be responsible for looking at a single use, if that's what it wants to do.